src/GeneralStateTestsFiller/stEIP3670/CreateTransactionEOF1InvalidOpcodesFiller.yml

```diff
@@ -0,0 +1,275 @@
+      - ':label inv_d0 :raw 0x68ef000101000200d00060005260096017f3'
+      - ':label inv_d1 :raw 0x68ef000101000200d10060005260096017f3'
+      - ':label inv_d2 :raw 0x68ef000101000200d20060005260096017f3'
+      - ':label inv_d3 :raw 0x68ef000101000200d30060005260096017f3'
+      - ':label inv_d4 :raw 0x68ef000101000200d40060005260096017f3'
+      - ':label inv_d5 :raw 0x68ef000101000200d50060005260096017f3'
+      - ':label inv_d6 :raw 0x68ef000101000200d60060005260096017f3'
+      - ':label inv_d7 :raw 0x68ef000101000200d70060005260096017f3'
+      - ':label inv_d8 :raw 0x68ef000101000200d80060005260096017f3'
+      - ':label inv_d9 :raw 0x68ef000101000200d90060005260096017f3'
+      - ':label inv_da :raw 0x68ef000101000200da0060005260096017f3'
+      - ':label inv_db :raw 0x68ef000101000200db0060005260096017f3'
+      - ':label inv_dc :raw 0x68ef000101000200dc0060005260096017f3'
+      - ':label inv_dd :raw 0x68ef000101000200dd0060005260096017f3'
+      - ':label inv_de :raw 0x68ef000101000200de0060005260096017f3'
+      - ':label inv_df :raw 0x68ef000101000200df0060005260096017f3'
+      - ':label inv_e0 :raw 0x68ef000101000200e00060005260096017f3'
+      - ':label inv_e1 :raw 0x68ef000101000200e10060005260096017f3'
+      - ':label inv_e2 :raw 0x68ef000101000200e20060005260096017f3'
+      - ':label inv_e3 :raw 0x68ef000101000200e30060005260096017f3'
+      - ':label inv_e4 :raw 0x68ef000101000200e40060005260096017f3'
+      - ':label inv_e5 :raw 0x68ef000101000200e50060005260096017f3'
+      - ':label inv_e6 :raw 0x68ef000101000200e60060005260096017f3'
+      - ':label inv_e7 :raw 0x68ef000101000200e70060005260096017f3'
+      - ':label inv_e8 :raw 0x68ef000101000200e80060005260096017f3'
+      - ':label inv_e9 :raw 0x68ef000101000200e90060005260096017f3'
+      - ':label inv_ea :raw 0x68ef000101000200ea0060005260096017f3'
+      - ':label inv_eb :raw 0x68ef000101000200eb0060005260096017f3'
+      - ':label inv_ec :raw 0x68ef000101000200ec0060005260096017f3'
+      - ':label inv_ed :raw 0x68ef000101000200ed0060005260096017f3'
+      - ':label inv_ee :raw 0x68ef000101000200ee0060005260096017f3'
+      - ':label inv_ef :raw 0x68ef000101000200ef0060005260096017f3'
+      - ':label inv_f2 :raw 0x68ef000101000200f20060005260096017f3'
+      - ':label inv_f6 :raw 0x68ef000101000200f60060005260096017f3'
+      - ':label inv_f7 :raw 0x68ef000101000200f70060005260096017f3'
+      - ':label inv_f8 :raw 0x68ef000101000200f80060005260096017f3'
+      - ':label inv_f9 :raw 0x68ef000101000200f90060005260096017f3'
+      - ':label inv_fb :raw 0x68ef000101000200fb0060005260096017f3'
+      - ':label inv_fc :raw 0x68ef000101000200fc0060005260096017f3'
+      - ':label inv_ff :raw 0x68ef000101000200ff0060005260096017f3'
+      # Invalid opcodes in initcode
+      # Initcode:
+      #   {
+      #     mstore(0, 0xef000101000100fe000000000000000000000000000000000000000000000000)
+      #     return(0, 8)
+      #     <invalid_opcode>
```

Comment:
Leave only `<invalid_opcode> STOP` in initcode, otherwise it's unreachable after RETURN

Reply:
Moved the invalid opcode to the beginning